### ISSUE TITLE
NTv2_transformation for Bosnia and Herzegovina

### ISSUE THREAD

**Issue by gnssgrid** [NaturalGIS/ntv2_transformations#49]
Hello
I want to perform Ntv2 transformation between old and new CRS. Unfortunately new CRS is not in EPSG database. Here are parameters for old and new crs:
Old crs:
+proj=tmerc +lat_0=0 +lon_0=15 +k=0.9999 +x_0=5500000 +y_0=0 +ellps=bessel
+proj=tmerc +lat_0=0 +lon_0=18 +k=0.9999 +x_0=6500000 +y_0=0 +ellps=bessel
+proj=tmerc +lat_0=0 +lon_0=21 +k=0.9999 +x_0=7500000 +y_0=0 +ellps=bessel 
New crs:
+proj=tmerc +lat_0=0 +lon_0=18 +k=0.9999 +x_0=500000 +y_0=0 +ellps=GRS80 
Unfortunately I can't share NTv2 file, because I developed inside my dissertation, and it can't be shared until I get permission from geodetic authority.
Is it possible to add this to your plugin? Thanks

**Comment by gioman** [comment]
@gnssgrid to be able to add this transformation is needed:

1) the NTV2 grid, that must have a clear license allowing redistribution

2) a few hours of work, that we won't do in our free time, feel free to hire us if you need (unless of course you want to send in a pull request with the necessary code changes, this is also welcome).

Cheers!

**Comment by gnssgrid** [comment]
@gioman 
Unfortunatelly I can't distribute NTv2 at this moment. Thanks anyway.

**Comment by gioman** [comment]
>  Unfortunatelly I can't distribute NTv2 at this moment. Thanks anyway.

@gnssgrid the plugin does not work without the binary ntv2 grid file.

**Comment by gnssgrid** [comment]
@gioman as I said I developed both ASCII and binary NTv2, but I can't distribute it because I don't have necessary permissions. 

Regards

**Comment by gioman** [comment]
> @gioman as I said I developed both ASCII and binary NTv2, but I can't distribute it because I don't have necessary permissions.

@gnssgrid yes I read that, I just said that there is no point in asking to add this transformation if the binary grid file cannot be made available (and redistributed).

**Comment by gnssgrid** [comment]
@gioman OK. I will let you know if I get permission to distribute it free.
